tests/jax/latest/vit.libsonnet

```diff
@@ -41,23 +41,30 @@ local tpus = import 'templates/tpus.libsonnet';
     rm .bash_logout
 
     pip install --upgrade pip
-    pip install --upgrade clu %(extraDeps)s
+
+    git clone https://github.com/google-research/vision_transformer.git
+    cd vision_transformer
+    pip install -r vit_jax/requirements.txt
+    cd ..
+    git clone https://github.com/google/flaxformer.git
+    cd flaxformer
+    pip3 install '.[testing]'
+    cd ../vision_transformer    
+
+    pip install --upgrade jaxlib jax flax clu %(extraDeps)s
```

Comment:
why are you installing jaxlib and jax twice? (that installLatest jax and jaxlib should already install jax and jaxlib?)

Reply:
I guess if in the requirements wrong versions are installed, maybe we can let the owners know that the requirements are wrong.